crates/rattler_conda_types/src/prefix_record.rs

```diff
@@ -234,8 +234,42 @@ impl PrefixRecord {
         path: impl AsRef<Path>,
         pretty: bool,
     ) -> Result<(), std::io::Error> {
-        let file = File::create(path.as_ref())?;
-        self.write_to(BufWriter::with_capacity(50 * 1024, file), pretty)
+        const BUFFER_SIZE: usize = 64 * 1024;
+
+        let path = path.as_ref();
+        let parent = path.parent().ok_or_else(|| {
```

Comment:
`parent()` will return `Some("")` if the file is not a absolute path. Is that okay?

Reply:
Yeah that should be OK since then it will jsut be a relative path (relative to CWD) and the rest of the types should also work with relative paths.